slides/08-Platform-Hub/1040-LetsOnboard.md

```diff
@@ -0,0 +1,10 @@
+## Let's start exploring!
+
+The following is your mission:
+
+1. Log into the platform hub and onboard yourself with github credentials (hint: you'll need a acp vpn profile for this, checkout https://access-acp.digital.homeoffice.gov.uk)
+2. Submit a new support request for adding a new project called developer-induction-\<fullname\> where \<fullname\> is your first and last name combined
```

Comment:
We should just have a single "Dev induction" project in which we add all participants in to (and then generate tokens for just the dev induction stuff). At the end of the dev induction we can clear this out.

Let's chat some more about this flow tomorrow.

Reply:
ok sounds good, the only caveat on this is I want user members to have the experience of being an admin and a non-admin on the hub. Though I don't want to make to task over complex but we'll discuss.